### PR TITLE
added support for gathering rke1 container logs

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

**Comment by cbron** [comment]
https://github.com/rancher/rancher/issues/27938

File: packages/rancher-logging/overlay/templates/loggings/rke1/logging-rke1.yaml [hunk 1]
```diff
@@ -4,7 +4,8 @@ kind: Logging
 metadata:
   name: {{ .Release.Name }}-rke
   namespace: {{ .Release.Namespace }}
-  labels: {{ include "logging-operator.labels" . | indent 4 }}
+  labels:
+{{ include "logging-operator.labels" . | indent 4 }}
```
Comment (rmweir) [inline packages/rancher-logging/overlay/templates/loggings/rke1/logging-rke1.yaml:8]: separate lines necessary?

File: packages/rancher-logging/overlay/templates/logging-containers-rke1.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        readOnly: true
+  fluentd:
+    image:
+      repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentd.repository }}
+      tag: {{ .Values.images.fluentd.tag }}
+    configReloaderImage:
+      repository: {{ template "system_default_registry" . }}{{ .Values.images.config_reloader.repository }}
+      tag: {{ .Values.images.config_reloader.tag }}
+    disablePvc: {{ .Values.disablePvc }}
+  {{- end }}
```
Comment (rmweir) [inline packages/rancher-logging/overlay/templates/logging-containers-rke1.yaml:28]: add newline here

File: packages/rancher-logging/overlay/templates/loggings/rke1/logging-rke1.yaml (hunk 1 above)
Comment (paynejacob) [reply packages/rancher-logging/overlay/templates/loggings/rke1/logging-rke1.yaml:8]: Yes I get template errors if it is on the sameline

**Pull request merged**